src/coreclr/jit/codegenarm64test.cpp

```diff
@@ -5912,6 +5912,100 @@ void CodeGen::genArm64EmitterUnitTestsSve()
     theEmitter->emitIns_R_PATTERN_I(INS_sve_incw, EA_SCALABLE, REG_V5, SVE_PATTERN_VL6, 16,
                                     INS_OPTS_SCALABLE_S); // INCW <Zdn>.S{, <pattern>{, MUL #<imm>}}
 
+    // IF_SVE_BO_1A
+    theEmitter->emitIns_R_PATTERN_I(INS_sve_sqdecb, EA_4BYTE, REG_R0, SVE_PATTERN_POW2,
```

Comment:
Here are below, the comment says `Xdn`, so this should be 8 bytes.
```suggestion
    theEmitter->emitIns_R_PATTERN_I(INS_sve_sqdecb, EA_8BYTE, REG_R0, SVE_PATTERN_POW2,
```

Reply:
I think that's the 32-bit pattern, right? The [docs](https://docsmirror.github.io/A64/2023-06/sqdecb_r_rs.html#iclass_32_fsreg) say the 64-bit pattern is `<Xdn>{, <pattern>{, MUL #<imm>}}`.